Specifications/Language/5_Grammar/QSharpParser.g4

```diff
@@ -227,6 +228,8 @@ expression
     | expression 'w/' expression '<-' expression
     ;
 
+sizeKey : Identifier {_localctx.getText() == "size"}?;
```

Comment:
Given the target language is Java, `{_localctx.getText() == "size"}?` should be `{_localctx.getText().equals("size")}?`

Reply:
Thank you!